### ISSUE TITLE
The program suddenly fails after one launch, but it can run in the Windows Sandbox.

### ISSUE THREAD

**Issue by jiut** [valinet/ExplorerPatcher#3572]
Win 11 Pro 23H2 22631.4037
EP 22621.3880.66.4

Reinstalled multiple times.

**Comment by pyrates999** [comment]
Please define what you mean it does not work.  What is it doing where it doesn't work?

Please specify if you are running on x86-64 or Arm.

Windows defender might be interfering here:

Set windows defender to exclude the following:
1. %ProgramFiles%\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher
3. Directory that you downloaded ep_setup.exe to

Then try to uninstall and reinstall EP using the latest pre-release.

If that doesn't fix it, in the EP properties application, go to **About** and click on **Restore default settings**.  Then reboot.

**Comment by jiut** [comment]
x64
AV off
No property option when right clicking taskbar.
I chose win10 style taskbar, start menu and explorer. The explorer and taskbar do not change. The start menu disappears.
When I open patcher menu through shortcut, it can only hide some icons on taskbar, other options do not work.

**Comment by pyrates999** [comment]
In the EP properties application, go to **About** and click on **Restore default settings**.  Then reboot.

Also check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.  Copy and paste the results here.

If you're referring to the icons in the system tray: You can drag and drop the icon in the system tray to the popup menu that you don't want shown.

that is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

**Comment by jiut** [comment]
I have tested all the options in order, from first to last, only options in the box can work, all the other options are non-functional.

When choosing win10 start menu, the start menu disappears.
Can't enable dump memory leaks,
Options with circle in the "Advanced" tab are not tested.

![1](https://github.com/user-attachments/assets/2a2363ca-517e-4cb4-9e9e-b4a8d9129d80)
![2](https://github.com/user-attachments/assets/27909804-9f76-46dc-b601-193241a71067)
![3](https://github.com/user-attachments/assets/4e0832d8-5318-4303-a14d-3b8b7052b72b)
![4](https://github.com/user-attachments/assets/13a8d5e5-527b-4f37-80f1-61c3e92e110b)
![5](https://github.com/user-attachments/assets/696fce58-d5d8-44c1-8fc4-5e09c0e5f1ae)
![6](https://github.com/user-attachments/assets/2833fd3a-8648-430a-aae5-9a821c30d74e)

**Comment by pyrates999** [comment]
can you show a screen shot after you run winver?

Your desktop windows version doesn't show the same version and build that you posted.

**Comment by jiut** [comment]
![image](https://github.com/user-attachments/assets/9f23ec49-687f-40cb-a41a-cb18ff4f1bf1)
indeed

**Comment by pyrates999** [comment]
Can you check if you got notifications that symbols were being downloaded, and then another notification that symbols were done being downloaded?

**Comment by jiut** [comment]
No, not even once, that's what I worried about. No notification at all. It seems that files cannot be downloaded.

**Comment by pyrates999** [comment]
If this is a work computer, the firewall might be blocking EP from downloading the symbols from Microsoft's servers.  Is this computer connected to the internet when you try to install EP?

**Comment by jiut** [comment]
It used to work fine, but after one startup, it stopped working. I just tried switching to a new user to install it, and it's still the same. However, I tried it in Windows Sandbox, and it runs normally. These are also mentioned in the title.

**Comment by pyrates999** [comment]
You may need to repair your windows install.  Please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by pyrates999** [comment]
what av did you disable by the way?

**Comment by pyrates999** [comment]
EP doesn't change file system permissions.

**Comment by pyrates999** [comment]
Whatever AV you use, exclude the following:

1. %ProgramFiles%\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher
3. Directory that you downloaded ep_setup.exe to

Future updates to EP won't be flagged then.

**Comment by jiut** [comment]
> EP doesn't change file system permissions.

I mean the AV changed the permissions of explorer.exe..

**Comment by pyrates999** [comment]
That I'm not sure of.  Repairing your windows install would fix that though.  Just uninstall all AV first before you do it.  And check if the AV has a stand alone uninstaller as the built in ones don't usually uninstall everything.

**Comment by jiut** [comment]
Thanks reinstallation can fix this.
